datafusion/expr/src/expr.rs

```diff
@@ -1032,6 +1033,88 @@ impl Expr {
     }
 }
 
+impl Expr {
+    // Operation methods from traits.
+
+    //
+    // Arithmetic operations
+    //
+
+    /// Return `self + other`
+    #[allow(clippy::should_implement_trait)]
```

Comment:
Clippy fires warnings because these methods have same name on trait methods in `std::ops`. Without these methods, user may require to include specific traits from `std::ops::*` in their source. Or, should we re-export these traits in prelude?